packages/terra-i18n-plugin/tests/i18n-plugin-spec.test.js

```diff
@@ -142,13 +143,13 @@ describe('i18n-aggregator-plugin', () => {
 
     it('should log console warning for missing tranlsation files', () => {
       expect(consoleMessage).toContain('Translation file en.json not found for');
-      expect(consoleMessage).toContain('/tests/fixtures/node_modules/fixtures2/translations');
+      expect(consoleMessage).toContain(path.join('tests', 'fixtures', 'node_modules', 'fixtures2', 'translations'));
       expect(SpyOnConsoleWarn).toHaveBeenCalled();
     });
 
     it('should fill the tranlsation files with the expected information for each locales', () => {
       supportedLocales.forEach((locale, index) => {
-        expect(outputtedFileName[index]).toContain(`aggregated-translations/${locale}.js`);
+        expect(outputtedFileName[index]).toContain(path.join('aggregated-translations', `${locale}.js`));
         expect(outputtedFileContent[index]).toContain(`react-intl/locale-data/${locale}`);
```

Comment:
This is the line that will fail if we were to do path.join because this will be part of the import statement.